samples/plugins/mysql_odbc/connection-dialog.tcd

```diff
@@ -0,0 +1,10 @@
+<connection-dialog class='mysql_odbc'>
+  <connection-config>
+    <authentication-mode value='Basic' />
+    <authentication-options>
+      <option name="UsernameAndPassword" default="true" />
+    </authentication-options>
+    <port-prompt value="Port: " default="3306" />
+    <show-ssl-checkbox value="true" />
```

Comment:
Double check this works as expected before including in a sample.  There is other setup in the connection string I believe.